### ISSUE TITLE
pretrained  models used

### ISSUE THREAD

**Issue by sorobedio** [NUS-HPC-AI-Lab/Neural-Network-Diffusion#16]
Hello, can i have the same pretrained models used such as resnet18 on cifar10, to imagenet-1k
Resnet50 and others pretrained models.

**Comment by 1zeryu** [comment]
Sorry that the checkpoints in our previous experiments have been deleted due to taking up too much storage space.
You can see if there are checkpoints you need on [torch/torchvision](https://pytorch.org/vision/stable/models.html) or other checkpoints website.
Hope this reply can help you.

**Comment by sorobedio** [comment]
thank you for your response. In fact the accuracies of the pretrained models on PyTorch page differ from the base accuracies you reported in your paper. So i wonder how you got those accuracy of the pretrained models.
thank you.
